specification/sql/resource-manager/readme.md

```diff
@@ -809,7 +809,8 @@ input-file:
 - ./Microsoft.Sql/preview/2021-02-01-preview/Usages.json
 - ./Microsoft.Sql/preview/2021-05-01-preview/LongTermRetentionBackups.json
 - ./Microsoft.Sql/preview/2021-05-01-preview/LongTermRetentionManagedInstanceBackups.json
-- ./Microsoft.Sql/preview/2021-05-01-preview/RestorableDroppedDatabases.json
+- ./Microsoft.Sql/preview/2021-05-01-preview/ManagedInstances.json
```

Comment:
Why do we have this one as duplicated ManagedInstances?

Reply:
Looks like leftover from rebase. Fixed it